hal/halx86/generic/buildtype.c

```diff
@@ -12,3 +12,9 @@
 /* GLOBALS ******************************************************************/
 
 const USHORT HalpBuildType = HAL_BUILD_TYPE;
+
+#ifdef CONFIG_SMP
+KIRQL HalpIrqlSynchLevel = (IPI_LEVEL - 2);
+#else
```

Comment:
You may want to update line 4 too.
https://github.com/reactos/reactos/blob/141b0a800a4c081a529766c7cec19b7ea690ca8e/hal/halx86/generic/buildtype.c#L4